v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -4,7 +4,11 @@ redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/ind
 ---
 
 This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm. 
```

Comment:
Pardon my stupidity. I was in the middle of a meeting while I was look at this and did not bother to think about checking out the branch with my changes. Fixed now.